deps/build.jl

```diff
@@ -1,14 +1,33 @@
 using Compat
 
-if haskey(ENV, "CONDA_JL_HOME")
-    ROOTENV = ENV["CONDA_JL_HOME"]
-elseif isfile("deps.jl")
-    include("deps.jl")
-else
-    ROOTENV = abspath(dirname(@__FILE__), "usr")
+module DefaultDeps
+    if isfile("deps.jl")
+        include("deps.jl")
```

Comment:
darn I thought this was a clever solution -- but clearly did not think it fully through.
Still simple patch is possible